src/function/numeric/solveODE.js

```diff
@@ -0,0 +1,254 @@
+import { isUnit } from '../../utils/is.js'
+import { factory } from '../../utils/factory.js'
+
+const name = 'solveODE'
+const dependencies = [
+  'typed',
+  'add',
+  'subtract',
+  'multiply',
+  'divide',
+  'max',
+  'map',
+  'abs',
+  'isPositive',
+  'isNegative',
+  'larger',
+  'smaller'
+]
+
+export const createSolveODE = /* #__PURE__ */ factory(name, dependencies, (
+  {
+    typed,
+    add,
+    subtract,
+    multiply,
+    divide,
+    max,
+    map,
+    abs,
+    isPositive,
+    isNegative,
+    larger,
+    smaller
+  }
+) => {
+  /**
+     * Numerical Integration of Ordinary Differential Equations
+     *
+     * Two variable step methods are provided:
+     * - "RK23": Bogacki–Shampine method
+     * - "RK45": Dormand-Prince method RK5(4)7M (default)
+     *
+     * The arguments are expected as follows.
```

Comment:
Can you also explain what the function returns (i.e. and object `{t, y}`)?

Reply:
I think it should be better now

Reply:
👍